src/items/class-features/armor_slot_(ex_exocortex).json

```diff
@@ -42,7 +42,7 @@
       "gmnotes": "",
       "short": "",
       "unidentified": "",
-      "value": "<p>&gt;@UUID[Compendium.sfrpg.class-features.z8iOan4RJXYvVhit]{Exocortex Mods (Ex)}</p>\n<p>Your exocortex gains an armor upgrade slot for an armor upgrade (see page 204). If you do not already have an armor upgrade to install, you must purchase one separately. You can install the armor upgrade only if your exocortex has enough open upgrade slots to meet the upgrade’s requirements. You can select this mod up to four times, each time adding an additional slot.</p>"
+      "value": "<p>&gt; @UUID[Compendium.sfrpg.class-features.z8iOan4RJXYvVhit]{Exocortex Mods (Ex)}</p>\n<p>Your exocortex gains an armor upgrade slot for an armor upgrade (see page 204). If you do not already have an armor upgrade to install, you must purchase one separately. You can install the armor upgrade only if your exocortex has enough open upgrade slots to meet the upgrade’s requirements. You can select this mod up to four times, each time adding an additional slot.</p>"
```

Comment:
Did you add these spaces after the `>`? No worries if not

Reply:
I did.  I thought you indicated to make there there was a space before the links.  I may have misunderstood.  I can fix if the > negates that need.

Reply:
The `>` is the exception to that unfortunately, there should be no space between the `>` and the link. Apologies.